### PR TITLE
Display real services in the deployment flow

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- use the df flag
- add a couple of services
- click deploy
- choose a cloud
- scroll down to the services section and check that your services are listed

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5461/
Test PASSed.

File: jujugui/static/gui/src/app/utils/changes-utils.js [hunk 1]
```diff
@@ -51,6 +51,28 @@ YUI.add('changes-utils', function(Y) {
   };
 
   /**
+    Return the counts for each type of ecs change.
+
+    @method getChangeCounts
+    @param {Object} services The list of services from the db.
+    @returns {Object} The ecs change counts.
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/utils/changes-utils.js]: These doc strings don't match the function.

**Comment by anthonydillon** [comment]
Code looks good with one small comment. :+1:

File: jujugui/static/gui/src/app/utils/changes-utils.js (hunk 1 above)
Comment (huwshimi) [inline jujugui/static/gui/src/app/utils/changes-utils.js]: Cripes, I'm terrible at remembering that!

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5462/
Test PASSed.

**Comment by huwshimi** [comment]
Thanks @anthonydillon :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**